ckanext/scheming/templates/scheming/snippets/render_fields.html

```diff
@@ -0,0 +1,30 @@
+{#}
+    fields - a list of scheming field dictionaries
+    data - form data fields
+    errors - A dict of errors for the fields
+    entity_type - entity type
+    object_type - object type
+    set_fields_defaults - flag to set the default field values
+{#}
+
+{% for field in fields if field.form_snippet is not none %}
+    {% if field.field_name not in data and set_fields_defaults %}
```

Comment:
All the other templates are indented with 2 spaces. Better to be consistent

Reply:
Sure, thanks